src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/wwwroot/swagger/swagger-utils.js

```diff
@@ -14,17 +17,44 @@ const initLoginForm = () => {
 
     overrideSwaggerAuthorizeEvent(swagger);
     overrideSwaggerLogoutEvent(swagger);
+    tryAuthorizeWithLocalData(swagger);
     showLoginUI(swagger);
 }
 
+const tryAuthorizeWithLocalData = (swagger) => {
+    if (isAuthorized(swagger))
+        return;
+
+    const token = getCookie(AUTHORIZATION_TOKEN_KEY);
+    if (!token)
+        return;
+
+    const authorizationObject = getAuthorizationRequestObject(token);
+    swagger.authActions.authorize(authorizationObject);
+}
+
 const overrideSwaggerAuthorizeEvent = (swagger) => {
     const auth = swagger.authActions.authorize;
-    swagger.authActions.authorize = (args) => attachReloadPageToEvent(swagger, auth, args);
+    swagger.authActions.authorize = async (args) => {
+        const result = await auth(args);
```

Comment:
plz remove the `auth` variable and use its value directly here.

Reply:
it is not possible because if I change this, the Authorize method becomes an infinite loop, and finally, a call stack error happen. this problem applies to the logout method too

Reply:
I don't understand, if you remove the variable the line becomes like this:

```js
const result = await swagger.authActions.authorize(args);
```

what makes it an infinite loop? did you test it out?

Reply:
Yes, I tested this and face the call stack error.

Reply:
I'm pretty sure this idea results into infinity loop @msynk

Reply:
oh I see, it's overriding the default functionality. ok then.